### ISSUE TITLE
All .pug files recompiled every time a .pug file is saved

### ISSUE THREAD

**Issue by joseph-d** [mrkmg/laravel-elixir-pug#8]
I'm using Laravel 5.3.8, Elixir 6, Node 6.5.0 and npm 3.10.3

I have been using Jade 1.11.0 and laravel-elixir-jade 0.2.4 in my project and it has been working well. All jade files in my search path are compiled properly when I first run `gulp` and then after that if I run `gulp watch` and change a jade file, only that single file is recompiled which is exactly how I want it.

However, I am trying to use pug with the same setup and I have installed the latest pug 2 (beta) and the latest laravel-elixir-pug. I have the following in my gulpfile:

```
        .pug({
            blade: true,
            search: '**/*.pug',
        })
```

The problem now, however, is that every time I edit a pug file, _all_ of the pug files in my project are recompiled rather than just the one I saved. This also has the knock-on effect of browsersync reloading once for each one of them every time I save a single pug file!

**Comment by mrkmg** [comment]
This should be fixed in 1.1.3. Please let me know if the changes fix your issue.

**Comment by joseph-d** [comment]
Works great, thank you!
